Sources/MapboxDirections/MapMatching/MatchResponse.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+class MatchResponse: Codable {
```

Comment:
`MatchResponse` seems removed in some other PR. I assume it means that that tracepoint filtering should occur in `MapMatchingResponse` instead?

Reply:
At the time this PR was originally written, MatchResponse interpreted Map Matching API responses as Map Matching API responses (MatchOptions→Match), whereas MapMatchingResponse interpreted Map Matching API responses as Directions API responses (MatchOptions→Route). It was confusing, so #406 eliminated MapMatchingResponse and renamed MatchResponse to MapMatchingResponse.